common/scripts/changelog/utils.mjs

```diff
@@ -1,11 +1,19 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+// Returns the filename in group1 of the regex match
+const CHANGE_FILE_REGEX = /(?:change|change-beta)\/(@azure-communication-react[^\/]*\.json)(?:.*)?$/;
+
 export function parseNewChangeFiles(stdout) {
-    // Split the output into lines
-    const lines = stdout.split('\n');
-    // Filter out the empty lines (`lines` list might have empty lines as part of the output)
-    const matches = lines.filter(line => line && line.toString().trim().length > 0);
+    // Split the output into lines and filter out the empty lines
+    const stdoutLines = stdout.split('\n').filter(line => line && line.toString().trim().length > 0);
+
     // Parse the change file name from the output (remove the status and return only file information)
-    return matches.map(line => line.split('\t')[1]);
-}
\ No newline at end of file
+    const matches = stdoutLines.map(line => {
+        const match = line.match(CHANGE_FILE_REGEX);
```

Comment:
As we try to match the changelog file name with the REGEX, we probably should also include a test where change or change-beta folder contains a file that is not changelog item

Reply:
Good idea, I'll add that now

Reply:
Added